crml/staking/src/lib.rs

```diff
@@ -1382,7 +1301,7 @@ decl_module! {
 		}
 
 		/// Force a current staker to become completely unstaked, immediately.
-		#[weight = T::WeightInfo::force_unstake()]
+		#[weight = T::WeightInfo::force_unstake(1_u32)]
```

Comment:
why 1?

Reply:
https://github.com/plugblockchain/plug-blockchain/blob/origin/spike/2.0.0/frame/staking/src/lib.rs#L1862
If you check the force_unstake
#[weight = T::WeightInfo::force_unstake(*num_slashing_spans)]
		fn force_unstake(origin, stash: T::AccountId, num_slashing_spans: u32) {

It uses num_slashing_spans which we don't have so I used 1

Reply:
https://github.com/plugblockchain/plug-blockchain/blob/origin/spike/2.0.0/frame/staking/src/default_weights.rs#L102
the parameter is getting multiplied so I kept it 1